pkgs/os-specific/linux/iputils/default.nix

```diff
@@ -44,6 +47,25 @@ in stdenv.mkDerivation rec {
   nativeBuildInputs = [ meson ninja pkgconfig gettext libxslt.bin docbook_xsl_ns ];
   buildInputs = [ libcap ]
     ++ optional (!stdenv.hostPlatform.isMusl) libidn2;
+  postInstall = ''
+    install -D /dev/stdin $apparmor/bin.ping <<EOF
+    include <tunables/global>
+    $out/bin/ping {
+      include <abstractions/base>
+      include <abstractions/consoles>
+      include <abstractions/nameservice>
+      include "${apparmorRulesFromClosure {}
+       ([libcap] ++ optional (!stdenv.hostPlatform.isMusl) libidn2)}"
```

Comment:
This could just be buildInputs, to dedup

Reply:
Indeed, but I don't think this a good habit to take and example to provide, because the dependencies of a package are not necessarily the same as the dependencies of a specific executable of this package. For instance here the `tftpd` in the same package does not depend on `libpcap` nor `libidn2` (according to `ldd`). Same could apply to `ping` in the future if another executable needs a `buildInputs` item not required by `ping`. This mismatch is [the main reason](https://github.com/NixOS/nixpkgs/pull/93457#issuecomment-705606683) why this PR moves AppArmor profiles from NixOS to Nixpkgs.